examples/services/eiav2/create_eia_service/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging"
+
+	"github.com/equinix/equinix-sdk-go/services/eiav2"
+	"github.com/equinix/oauth2-go"
+)
+
+func main() {
+	ctx := context.Background()
+	clientId := os.Getenv("EQUINIX_API_CLIENTID")
+	clientSecret := os.Getenv("EQUINIX_API_CLIENTSECRET")
+	baseURL := "https://api.equinix.com"
+	authConfig := oauth2.Config{
+		ClientID:     clientId,
+		ClientSecret: clientSecret,
+		BaseURL:      baseURL,
+	}
+	authClient := authConfig.New(ctx)
+	authClient.Transport = logging.NewTransport("Equinix", authClient.Transport)
+
+	transport := logging.NewTransport("Equinix Fabric (eiav2)", authClient.Transport)
```

Comment:
Similar to my comment on the Fabric example, we should strive to keep these examples tooling-agnostic; `logging` is a package that is part of terraform, but users might use the Go SDK in other places where they don't need to pull in terraform components.

Reply:
Example updated 👍 

This PR should be ready to go now.